starfish/pipeline/filter/richardson_lucy_deconvolution.py

```diff
@@ -0,0 +1,90 @@
+from functools import partial
+from typing import Callable
+
+import numpy as np
+from skimage import restoration
+
+from starfish.io import Stack
+from ._base import FilterAlgorithmBase
+from .util import gaussian_kernel
+
+
+class DeconvolvePSF(FilterAlgorithmBase):
+
+    def __init__(self, num_iter: int, sigma: int, clip: bool, **kwargs):
+        """Deconvolve a point spread function
+
+        Note that the default parameters are highly optimized for the MERFISH use case and that num_iter is a
+        very important parameter that requires careful optimization.
+
+        Parameters
+        ----------
+        num_iter : int
+            number of iterations to run
+        sigma : int
+            standard deviation of the gaussian kernel used to construct the point spread function
+        clip : bool (default = False)
```

Comment:
This is inconsistent with the function signature.

Reply:
will fix.